shared/actions/fs/platform-specific.desktop.tsx

```diff
@@ -236,17 +234,21 @@ const fuseInstallResultIsKextPermissionError = (result: RPCTypes.InstallResult):
     c => c.name === 'fuse' && c.exitCode === Constants.ExitCodeFuseKextPermissionError
   ) !== -1
 
-const driverEnableFuse = (_, action: FsGen.DriverEnablePayload): Promise<Saga.MaybeAction> =>
-  RPCTypes.installInstallFuseRpcPromise().then((result: RPCTypes.InstallResult) =>
-    fuseInstallResultIsKextPermissionError(result)
-      ? [
+const driverEnableFuse = (_, action: FsGen.DriverEnablePayload): Saga.ChainActionReturn =>
+  new Promise(resolve => {
```

Comment:
I don't think you need this `Promise`, removing it and adding `return`s gives no errors for me.

Reply:
i converted it to async and cleaned that up, tnx!